staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/kmsv2/envelope.go

```diff
@@ -165,6 +178,11 @@ func (t *envelopeTransformer) TransformToStorage(ctx context.Context, data []byt
 	statusKeyID, err := t.keyIDGetter(ctx)
 	if err == nil && encObject.KeyID != statusKeyID {
 		klog.V(2).InfoS("observed different key IDs when encrypting content using kms v2 envelope service", "uid", uid, "objectKeyID", encObject.KeyID, "statusKeyID", statusKeyID)
```

Comment:
should we add the provider name to the logs here too? 

Now that we have the provider name as part of encrypt/decrypt call, i think it might be good for us to include that in all the logs as part of the package. Not a blocker for this PR.